src/components/tweet-link.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import {sample} from 'lodash'
+import {displayListify} from '../../shared/utils'
+import shortenEpisodeUrl from '../../other/shorten-episode-url/logic'
+
+export default TweetLink
+
+function addRandomAwesomeEmojiTo(message) {
+  const emoji = sample(['✨', '💥', '🎉', '🚀', '🌟', '🎊', '👍', '💯', '👏', '🙌', '😎', '🔥', '😻'])
+  const AVAILABLE_TWEET_LENGTH = 140
+  if (AVAILABLE_TWEET_LENGTH - message.length >= emoji.length) {
+    return `${message} ${emoji}`
+  }
+
+  return message
+}
+
+function TweetLink(props) {
+  const {
+    episode,
+    guests,
+    isPast,
+  } = props
+
+  const title = episode.titleHTML.__html
+  const date = episode.dateDisplay
+  const shortUrl = shortenEpisodeUrl(episode, true)
```

Comment:
Don't use `shortenEpisodeUrl`, instead you can just get the existing shortUrl off of the `episode`:

```javascript
const {shortUrl} = episode
```

👍